pyanaconda/payload/__init__.py

```diff
@@ -472,7 +472,7 @@ def _setup_device(device, mountpoint):
             device.format.setup(mountpoint=mountpoint)
```

Comment:
Again please rename the commit messages from ``Use the function...`` to ``Add a new function...``

Reply:
Fixed.